### PR TITLE
New create petition steps sequence

### PR DESCRIPTION
**Pull request by mybits**
New petition create steps sequence:
create petition --> check your petition --> sign your petition --> make sure this is right (email replay)

PT: https://www.pivotaltracker.com/n/projects/307301/stories/96866290

### CODE REVIEW

File: app/models/staged/petition_creator/replay_petition.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 module Staged
   module PetitionCreator
     class ReplayPetition < Staged::Base::Petition
-      include Staged::PetitionCreator::HasCreatorSignature
 
       class CreatorSignature < Staged::Base::CreatorSignature
```
Comment (h-lame) [inline app/models/staged/petition_creator/replay_petition.rb]: If we don't include the ``Staged::PetitionCreator::HasCreatorSignature`` anymore, we also don't need the ``Staged::PetitionCreator::ReplayPetition::CreatorSignature``.

**Comment by h-lame** [comment]
One more thing that can be deleted, but :+1: on the rest.

File: app/models/staged/petition_creator/replay_petition.rb (hunk 1 above)
Comment (mybits) [inline app/models/staged/petition_creator/replay_petition.rb]: Oh yes, absolutely! Will remove

**Comment by mybits** [comment]
thanks @h-lame! redundant class removed here: https://github.com/alphagov/e-petitions/commit/aa2b741c9802e6a4490d63eb861b8f9d82623322

**Comment by otlaitil** [comment]
:+1:

**Pull request merged**